db/migrate/20200514092553_move_katello_fields_from_hostgroups.katello.rb

```diff
@@ -0,0 +1,35 @@
+class MoveKatelloFieldsFromHostgroups < ActiveRecord::Migration[6.0]
+  def up
+    copy_data_from_hostgroup
+
+    change_table :hostgroups do |table|
+      table.remove(
+        :content_source_id,
+        :kickstart_repository_id,
+        :content_view_id,
+        :lifecycle_environment_id
+      )
+    end
+  end
+
+  def down
+  end
+
+  def copy_data_from_hostgroup
```

Comment:
I think we should have a test for this migration. May have found a problem:

hostgroup prior to migration:
```
[3] pry(main)> hg.content_source_id
=> 1
[4] pry(main)> hg.kickstart_repository_id
=> nil
[5] pry(main)> hg.content_view_id
=> 3
[6] pry(main)> hg.lifecycle_environment_id
=> 1
```
hostgroup post migration:

```
[2] pry(main)> hg.lifecycle_environment_id
=> nil
[3] pry(main)> hg.content_view_id
=> nil
[4] pry(main)> hg.kickstart_repository_id
=> nil
[5] pry(main)> hg.content_source_id
=> nil
[6] pry(main)> hg.content_facet
=> nil
```

Additionally:
```
[14] pry(main)> ::Hostgroup.all.size
=> 2
[15] pry(main)> Katello::Hostgroup::ContentFacet.all
=> []
```

Reply:
Fixed the migration - changes to the model prevented it from working properly and tested it locally.
I have no idea how to test this code - I did it with running queries directly against the DB.